packages/backend/src/graphql/resolvers/walletAddressKey.ts

```diff
@@ -4,10 +4,46 @@ import {
   WalletAddressKey as SchemaWalletAddressKey,
   Alg,
   Kty,
-  Crv
+  Crv,
+  WalletAddressResolvers
 } from '../generated/graphql'
 import { ApolloContext } from '../../app'
 import { WalletAddressKey } from '../../open_payments/wallet_address/key/model'
+import { Pagination } from '../../shared/baseModel'
+import { getPageInfo } from '../../shared/pagination'
+
+export const getWalletAddressKeys: WalletAddressResolvers<ApolloContext>['walletAddressKeys'] =
```

Comment:
we should write a test for this. see if you can repurpose `getPageTests` (`packages/backend/src/shared/baseModel.test.ts`) maybe.

Would be good to test `getWalletAddressKeyPage` too.

Reply:
Yes, I was able to use `getPageTests` and I also wrote tests for `getWalletAddressKeyPage`